code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -86,4 +86,5 @@
 	var/hal_screwydoll
 	/// When an braindead player has their equipment fiddled with, we log that info here for when they come back so they know who took their ID while they were DC'd for 30 seconds
 	var/list/afk_thefts
-
+	/// This supresses the "dosen't appear to be himself" examine text for if the mob is run by an AI controller. Should be used on any NPC human subtypes. Monkeys are the prime example.
+	var/non_player_species = FALSE
```

Comment:
Fair point, but it will take an extra check on the examine to avoid potential runtime errors because this examine proc is on /mob/living/carbon/human. Not /mob/living/carbon/human/species/monkey.

Any monkeys that AREN'T from human/species/monkey, would not have this problem anyway because they don't inherit the examine text from /mob/living/carbon/human. which is the whole problem in the first place.

Also, post the aforementioned refractor, all monkeys are from human/species/monkey to my understanding anyways.

Reply:
Ok nevermind I get entirely what you mean.